org.gnome.Evince.json

```diff
@@ -134,9 +157,15 @@
             ],
             "sources": [
                 {
-                    "type": "archive",
-                    "url": "https://download.gnome.org/sources/gspell/1.8/gspell-1.8.4.tar.xz",
-                    "sha256": "cf4d16a716e813449bd631405dc1001ea89537b8cdae2b8abfb3999212bd43b4"
+                    "type": "git",
```

Comment:
This should use the stable version. Not the development one.

Reply:
Ditto. Why is it the development version when they tagged it ? Or maybe they have a different release scheme?

Reply:
The gnome modules should definitely wait until the gnome checker lands otherwise you will have to deal with the stable/instable release schemes.

Reply:
> Ditto. Why is it the development version when they tagged it ? Or maybe they have a different release scheme?

From gspell Readme:

> gspell follows the GNOME release schedule and uses the same versioning scheme:
>
>    https://developer.gnome.org/programming-guidelines/stable/versioning.html.en
>
> For example the 1.7.x versions are unstable, and the 1.8.x versions are stable.

You were changing 1.8.4 (stable) by 1.9.1 (unstable). Please avoid that unless there is a good reason to do so.